recipes/pslibrary/tests.sh

```diff
@@ -0,0 +1,1194 @@
+test -f ${PREFIX}/share/pslibrary/rel-pbe/PSEUDOPOTENTIALS/Ta.rel-pbe-spfn-rrkjus_psl.1.0.0.UPF
```

Comment:
Maybe here you can do a glob for all *.UPF, no?

Reply:
Does this help? I thought the idea is to confirm that all files are written. If I just use find or glob than it only identifies the ones which are written, but I do not notice missing ones.

Reply:
I'm not familiar with this package to be honest, so maybe my request does not make sense. Checking for each file is not sustainable and hard to maintain, but if it is necessary let's do it.

Reply:
It is similar like https://github.com/conda-forge/gpaw-data-feedstock and https://github.com/conda-forge/iprpy-data-feedstock

Reply:
My question is more like: "Is there a scenario where one of those files would not be created?"
If the expected result are all or nothing than a glob is fine. If the build can fail a single file and move on to a successful build, then there is not much we can do.

Reply:
I guess for now I would test all and then update the tests as soon as I have some experience with the changes switching from one version to the next.